pkg/ccl/importccl/import_stmt_test.go

```diff
@@ -4038,6 +4011,20 @@ func TestImportMysql(t *testing.T) {
 			}
 		})
 	}
+	t.Run("import-into-not-supported", func(t *testing.T) {
+		data := `INSERT INTO t VALUES (1, 2), (3, 4)`
+		srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			if r.Method == "GET" {
+				_, _ = w.Write([]byte(data))
+			}
+		}))
+		defer srv.Close()
+		defer sqlDB.Exec(t, "DROP TABLE t")
+		sqlDB.Exec(t, "CREATE TABLE t (a INT, b INT)")
+		sqlDB.ExpectErr(t,
+			"MYSQLDUMP file format is currently unsupported by IMPORT INTO",
+			fmt.Sprintf(`IMPORT INTO t (a, b) MYSQLDUMP DATA (%q)`, srv.URL))
+	})
 }
 
 func TestImportMysqlOutfile(t *testing.T) {
```

Comment:
Not related to your change, but since you're here can you rename this test to `TestImportDelimited`?

Reply:
Done (along with the change that moves the `MySQLDump` out to be a standalone test, named `ImportIntoMySQL`).